README.md

```diff
@@ -123,6 +126,10 @@ require'compe'.setup {
   max_kind_width = 100;
   max_menu_width = 100;
 
+  documentation = {
+    show = 'enable';
+  };
```

Comment:
Sorry for my poor English.

My intention is we should provide only `let g:compe.documentation = v:true | v:false` for now.